pkg/rslog/debug.go

```diff
@@ -60,21 +60,16 @@ func InitDebugLogs(regions []ProductRegion) {
 
 	// Reset enabled regions on each call.
 	regionsEnabled = make(map[ProductRegion]bool)
+	lgr := DefaultLogger()
```

Comment:
I guess we have to instantiate a new logger here because we don't have the main one instantiated yet, right?

Reply:
The `DefaultLogger()` func calls a method that sets a default logger if it doesn't exist yet. https://github.com/rstudio/platform-lib/blob/6838bcc0d0594953823755e3029f9d199d834fed/pkg/rslog/logger_impl.go#L217-L234

Reply:
Will this have any effect on using a non-default logger factory? I don't think so, but wanted to be sure.

Reply:
Also, it looks like there's a test failure.

Reply:
Not a direct impact, but initializing debug logs now will set the default factory if it doesn't exist. That why the test fails, due to the order of init debug regions and then setting the test factory.